style.css

```diff
@@ -0,0 +1,247 @@
+
+html {
+  padding: 0;
+  margin: 0;
+  font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
+  list-style: none;
+  color: white;
+}
+
+.screen {
+  position: relative;
+  width: 375px;
+  height: 5415px;
+  background: #191765;
+  display: inline-block;
+}
+
+.app-bar-compact {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.status-bar {
+  position: absolute;
+  width: 375px;
+  height: 44px;
+  left: calc(50% - 375px / 2);
+  top: 0;
+}
+
+.header {
+  width: 100;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.header-items {
+  width: 375px;
+  height: 92px;
+  position: absolute;
+  padding-top: 0;
+  margin-top: 0;
+}
+
+.my-logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 105px;
+  height: 36px;
+  left: 12px;
+  top: calc(50% - 36px / 2);
+}
+
+.logo-text {
+  width: 81px;
+  height: 16px;
+  font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
```

Comment:
- [ ] Kindly try to follow the Figma design as much as possible, please use the same font and the same margins used in Figma

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185494440-da456934-1fcc-455a-b4c4-44c67e60fb7c.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/185494495-543e9f93-cba5-4c7b-ac5c-476228041d17.png)